ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -179,7 +167,9 @@ conformance_test(
         "--daml-lf-min-version-1.14-unsafe",
     ],
     test_tool_args = [
-        "--exclude=CommandDeduplicationIT",  # It's a KV ledger so it needs the KV variant,
+        "--additional=MultiPartySubmissionIT",
```

Comment:
Can you please order the test flags? (e.g. first `additional` and then `exclude`).

Reply:
In this case or in general? I chose to keep (test-wise) related args together even though it's true that exclusions are applied last.

Reply:
I meant in general. But I see your point as well, so I don't have a strong opinion regarding it.

Reply:
It was actually possible to do so without separating related tests cd4cf62